cli/cmd/release_create.go

```diff
@@ -172,8 +177,9 @@ Prepared to create release with defaults:
     version         %q
     release-notes   %q
     ensure-channel  %t
+    lint-release    %t
```

Comment:
would it make sense to also show the `fail-on` value here?

Reply:
Possibly if lint-release is true. Otherwise fail-on is ignored.

Reply:
I like that idea